crates/redis_interface/src/types.rs

```diff
@@ -57,6 +57,7 @@ pub struct RedisSettings {
     pub max_in_flight_commands: u64,
     pub default_command_timeout: u64,
     pub max_feed_count: u64,
+    pub unresponsive_timeout: u64,
```

Comment:
Include this in `config.example.toml` with a comment, also include the unit.

And add this in the other files like `development.toml`, `docker_compose.toml` and `loadtest/config/development.toml`.

Reply:
Done